chainerrl/agents/pcl.py

```diff
@@ -137,7 +137,7 @@ def __init__(self, model, optimizer,
         self.pi_loss_coef = pi_loss_coef
         self.v_loss_coef = v_loss_coef
         self.rollout_len = rollout_len
-        self.batchsize = batchsize
+        self.batchsize = self.xp.int32(batchsize)
```

Comment:
I guess this is a fix for newint, but it's not clear from the code. Can you add a comment?